packages/grid/_modules_/grid/components/panel/GridPanel.tsx

```diff
@@ -76,7 +78,6 @@ const GridPanel = React.forwardRef<HTMLDivElement, GridPanelProps>((props, ref)
       ref={ref}
       placement="bottom-start"
       className={clsx(className, classes.panel)}
```

Comment:
The `as` prop is missing as well as `rootProps.componentsProps.panel`.

Reply:
I can't add them in the `GridPanel` directly because the `useGridRootProps` can be used in components outside of `DataGrid`. That's why I had to pass it down from the parent. The same applies to the other comment https://github.com/mui-org/material-ui-x/pull/3490#discussion_r779507845

```
Error: Uncaught Error: MUI: useGridRootProps should only be used inside the DataGrid/DataGridPro component.

Reply:
We had a discussion about that in #2522.

Currently the `GridPanel` already depends on another context: `GridApiContext`. We could also use `useGridRootProps` but that would be a breaking change. So I'm OK to not do that for now.

Reply:
Ok so if we are postponing this and if this is the only thing left can we proceed with merging this PR?